docs/workflow/trimming/feature-switches.md

```diff
@@ -0,0 +1,56 @@
+# Libraries Feature Switches
+
+Starting with .NET 5 there are several [feature-switches](https://github.com/dotnet/designs/blob/master/accepted/2020/feature-switch.md) available which
+can be used to control the size of the final binary. They are available in all
+configurations but their defaults might vary as any SDK can set the defaults differently.
+
+## Available Feature Switches
+
+| MSBuild property name | C# property name | Description |
+|-|-|-|
+| EnableUnsafeUTF7Encoding | System.Text.Encoding.EnableUnsafeUTF7Encoding |  Insecure UTF-7 encoding is trimmed when set to false |
+| EventSourceSupport | System.Diagnostics.Tracing.EventSource.IsSupported | Any EventSource related code or logic is trimmed when set to false |
+| InvariantGlobalization | System.Globalization.Invariant | All globalization specific code and data is trimmed when set to true |
+| UseSystemResourceKeys | System.Resources.UseSystemResourceKeys |  Any localizable resources for system assemblies is trimmed when set to true |
+| - | System.Diagnostics.Debugger.IsSupported | Any dependency that enables better debugging experience to be trimmed when set to false |
```

Comment:
The MSBuild property name here is `DebuggerSupport`. We already have it for the removing symbols in the SDK, we just need to add the `RuntimeHostConfigurationOption` in the SDK.

https://github.com/dotnet/sdk/blob/3b1ad2f01a20cd45a409ab4eed7821b8a631eb11/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets#L142